docs/datamodel/scalars/datetime.rst

```diff
@@ -1,28 +1,56 @@
 .. _ref_datamodel_scalars_datetime:
 
-Date/Time Types
-===============
+Date and Time
+=============
 
 .. eql:type:: std::datetime
 
     A type representing date, time, and time zone.
 
-.. eql:type:: std::date
+.. eql:type:: std::naive_datetime
 
-    A type representing date and time zone.
+    A type representing date and time without time zone.
 
-.. eql:type:: std::time
+.. eql:type:: std::naive_date
 
-    A type representing time and time zone.
+    A type representing date without time zone.
+
+.. eql:type:: std::naive_time
+
+    A type representing time without time zone.
 
 .. eql:type:: std::timedelta
 
     A type representing a relative time interval.
 
-    The time interval can be specified in terms of microseconds,
-    milliseconds, seconds, minutes, hours, days, weeks, months, years,
-    decades, centuries, millennia, e.g.:
+    The time interval can be specified in terms of *microseconds*,
+    *milliseconds*, *seconds*, *minutes*, *hours*, *days*, *weeks*,
+    *months*, *years*, *decades*, *centuries*, *millennia*, e.g.:
 
     .. code-block:: edgeql
 
         SELECT <timedelta>'2.3 millennia 3 weeks';
+
+    It's worth noting that time intervals are inherently ambiguous
```

Comment:
I'd explain why it is ambiguous (months have different number of days in them)

Reply:
I've updated that section.